### DISCUSSION TITLE
Possible to override commiter email/name to show it's verified in Codespaces?

### DISCUSSION THREAD

**Discussion by ajhalili2006** [community/community#18640] [Codespaces]
I enabled commit signing for Codespaces on some repos and also vigilant mode to force showing verification status across all commits I did, but the GitHub UI still says that it's unverified due to the fact that the committer is `GitHub <noreply@github.com>`[^1]. Is it possible to override `GIT_COMMITTER_EMAIL`/`GIT_COMMITTER_NAME` variable through encrypted secrets?

![image](https://user-images.githubusercontent.com/34998342/173762908-7e5b4510-13c1-4f22-b731-27851e196d24.png)

[^1]: The screenshot in this post is taken from https://github.com/RecapTime/content-warnings-radar-links/commit/fb25b89a388dd57ddde1ac767f4759a25272056f

**Comment by BurAndBY** [comment]
You need to have a GPG certificate - https://docs.github.com/en/codespaces/managing-your-codespaces/managing-gpg-verification-for-codespaces

**Comment by tnir** [comment]
@ajhalili2006 I failed to do so last year as reported in https://github.community/t/failed-to-create-a-codespace/200326
